spec/request/health_care_applications_request_spec.rb

```diff
@@ -111,10 +111,14 @@
           allow_any_instance_of(HCA::Service).to receive(:post) do
             raise error
           end
+          Settings.sentry.dsn = 'asdf'
+        end
+        after do
+          Settings.sentry.dsn = nil
         end
 
         it 'should render error message' do
-          expect(Raven).to receive(:capture_exception).with(error).once
+          expect(Raven).to receive(:capture_exception).with(error).twice
```

Comment:
being called twice is the desired behavior, but only because we are rendering errors by raising BackendServiceException. the sentry logging of BackendServiceException doesn't include the stack trace for the client error so i have to capture that.